db/migrate/20210617203927_migrate_admin_tax_amounts.rb

```diff
@@ -0,0 +1,52 @@
+class MigrateAdminTaxAmounts < ActiveRecord::Migration[6.0]
+  class Spree::Adjustment < ApplicationRecord
+    belongs_to :originator, polymorphic: true
```

Comment:
I don't see this relationship used. Would it be easier to omit this line and see `originator_id` and `originator_type` as simple attributes?

Reply:
I changed this migration a couple of times whilst working on it. I'll see if it can be tidied up a bit :ok_hand: